autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/FlowCommunicator.java

```diff
@@ -0,0 +1,28 @@
+package com.sequenceiq.periscope.monitor.handler;
+
+import javax.inject.Inject;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.client.CloudbreakInternalCrnClient;
+import com.sequenceiq.flow.api.model.FlowCheckResponse;
+import com.sequenceiq.periscope.aspects.RequestLogging;
+
+@Component
+public class FlowCommunicator {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(FlowCommunicator.class);
+
+    @Inject
+    private CloudbreakInternalCrnClient internalCrnClient;
+
+    @Inject
+    private RequestLogging requestLogging;
+
+    public FlowCheckResponse hasFlowRunningByChainId(String flowId) {
+        return requestLogging.logResponseTime(() -> internalCrnClient.withInternalCrn().flowEndpoint().hasFlowRunningByChainId(flowId),
```

Comment:
Is this using the correct APIs?
@vivprabu added an endTime to the flowLog APIs via https://github.com/hortonworks/cloudbreak/pull/13563 . We could either use those - or wait for the endTime to be available on the current API being used.